homeassistant/components/deconz/cover.py

```diff
@@ -85,52 +98,54 @@ def __init__(self, device, gateway):
         self._attr_device_class = DEVICE_CLASS.get(self._device.type)
 
     @property
-    def current_cover_position(self):
+    def current_cover_position(self) -> int:
         """Return the current position of the cover."""
+        assert isinstance(self._device.lift, int)
```

Comment:
Change of behaviour? (See comment above)

Reply:
Same response as above

Reply:
Removed assert